README.md

```diff
@@ -1,3 +1,66 @@
+
+# ShotCallers - Redefining Your Basketball Experience
+
+## Overview
+Welcome to ShotCallers, a revolutionary application designed by Team Shot Callers from the Learn Academy Golf cohort 2023-2024. Comprising Michael Reyes, Jeffrey Guzman, Eric McKee, and Franz Marte, our team envisions changing the landscape of basketball experiences.
+
+### Meet Our Team:
+- Michael Reyes
+- Jeffrey Guzman
+- Eric McKee
+- Franz Marte
+
+## Our Mission:
+ShotCallers transcends being just an app; it embodies a vision meticulously brought to life by Team Shot Callers. Our mission is clear: to create a platform that seamlessly connects basketball enthusiasts in real-time, enriching the experience for those in search of spontaneous pickup games.
+
+## Why Choose ShotCallers:
+1. **Real-Time Court Map:** Effortlessly explore nearby basketball games or discover empty courts instantly with our innovative real-time court map.
+2. **Spontaneous Connections:** Foster camaraderie among basketball enthusiasts by making every pickup game a memorable experience.
+3. **User-Centric Design:** Our app is thoughtfully crafted, ensuring that finding and participating in pickup games is not only easy but also enjoyable. We prioritize user experience to enhance your basketball journey.
+
+## Running the Project:
+To get ShotCallers up and running, follow these steps:
+
+1. Clone the repository: `git clone https://github.com/ShotCallers/ShotCallers-App.git`
+2. Navigate to the project directory: `cd ShotCallers-App`
+3. Install dependencies: `npm install` or `yarn`
+4. Run the app: `npm start` or `yarn start`
```

Comment:
I would just put yarn here.  This will lead to less conflicts and if a person insists on using npm as a package manager, they will know the drill.